src/main/java/at/hannibal2/skyhanni/features/combat/FlareDisplay.kt

```diff
@@ -177,6 +190,21 @@ object FlareDisplay {
         }
     }
 
+    @SubscribeEvent
+    fun onRender(event: GuiRenderEvent.GuiOverlayRenderEvent) {
+        if (!config.flashScreen || !activeWarning) return
```

Comment:
check if the main fature is enabled or make so that activeWarning gets disabled if the feature gets disabled while the warning is active